### PR TITLE
Make ci bintray subject as a parameter

### PR DESCRIPTION
**Pull request by PengTian0**
CI_BINATRAY was a staging Bintray (rackhd-mirror bintray) , which saves the deb packeages generated in deb-build step in pipeline, and those deb will be consume in latter ova-build/vagrant-build/docker-build steps . 
Those deb will NOT be publish to official Bintray (rackhd bintray) until all tests pass( FIT/post-test..).
so this PR is to make the "rackhd-mirror" obtained from the ENV Variable

### CODE REVIEW

File: jobs/build_debian/build_debian.sh [hunk 1]
```diff
@@ -12,8 +12,8 @@ curl --user $BINTRAY_CREDS -L "$MANIFEST_FILE_URL" -o rackhd-manifest
 --force \
 --is-official-release $IS_OFFICIAL_RELEASE \
 --bintray-credential BINTRAY_CREDS \
---bintray-subject $BINTRAY_SUBJECT \
---bintray-repo $BINTRAY_REPO
+--bintray-subject $CI_BINTRAY_SUBJECT \
```
Comment (panpan0000) [inline jobs/build_debian/build_debian.sh]: why ?

File: jobs/build_docker/build_docker.groovy [hunk 2]
```diff
@@ -3,6 +3,13 @@ node(build_docker_node){
         withEnv([
             "MANIFEST_FILE_URL=${env.MANIFEST_FILE_URL}",
             "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
+            "BINTRAY_SUBJECT=${env.BINTRAY_SUBJECT}",
```
Comment (panpan0000) [inline jobs/build_docker/build_docker.groovy:6]: I thought docker are build from src code , but I just found it apt-get install on-imagebiulder for static file. so I can understand this change now.

**Comment by panpan0000** [comment]
👍 
I proposed you added some description:
```
CI_BINATRAY was a staging Bintray (rackhd-mirror bintray) , which saves the deb packeages generated in deb-build step in pipeline, and those deb will be consume in latter ova-build/vagrant-build/docker-build steps . 
those deb will NOT be publish to official Bintray (rackhd bintray) until all tests pass( FIT/post-test..).
so this PR is to make the "rackhd-mirror" obtained from the ENV Variable 
```

**Comment by PengTian0** [comment]
test this please

**Review by anhou — APPROVED** [review]

**Pull request merged**